CollAction/Controllers/ProjectsController.cs

```diff
@@ -311,6 +317,25 @@ public IActionResult Embed()
             }
         }
 
+        [Authorize]
+        [HttpGet]
+        public IActionResult ThankYouCommit(string name)
+        {
+            if(name != null)
+            {
+                CommitProjectViewModel model = new CommitProjectViewModel()
+                {
+                    ProjectName = name
+                };
+                return View("ThankYouCommit", model);
```

Comment:
nameof(ThankYouCommit) would be robust again action renames.

Reply:
Fixed in 0544b69